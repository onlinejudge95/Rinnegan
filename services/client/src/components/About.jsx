import React from "react";
import { Container, Divider, Header } from "semantic-ui-react";

const About = () => {
  return (
    <main className="content">
      <Container>
        <div class="ui relaxed divided padded full grid">
          <div class="row">
            <div class="column">
              <Header as="h1">Rinnegan</Header>
            </div>
          </div>
          <Divider />
          <div class="row">
            <div class="wide column">
              <Container>
                <p>
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed
                  ac libero et enim fringilla viverra non et leo. Sed vitae
                  felis diam. Duis tempus ex sit amet diam feugiat mattis.
                  Aliquam fermentum, est id fermentum blandit, nisi tortor
                  elementum tortor, a tincidunt mi dolor ut lacus. Mauris
                  auctor, sem at dignissim efficitur, justo diam pellentesque
                  urna, sit amet ultricies leo tellus ut nulla. Mauris interdum
                  massa elit. Maecenas vel odio id nulla convallis dignissim.
                  Praesent diam orci, feugiat id gravida eget, eleifend
                  hendrerit massa. Morbi congue, nulla a dapibus tempus, sem
                  tellus iaculis nunc, et tincidunt lectus dui ut mauris.
                </p>
                <p>
                  Aenean finibus mollis mauris, vitae dignissim nunc fringilla
                  sed. Donec urna tortor, consequat quis diam in, condimentum
                  ornare mi. Cras maximus ex diam, sit amet bibendum ligula
                  molestie vitae. Integer a odio hendrerit, bibendum lorem vel,
                  tincidunt lacus. Praesent eu nunc ac enim fringilla dignissim
                  eu vel urna. Nulla ac commodo dolor, sed mollis nisi. Proin ac
                  posuere libero. Vivamus ultricies erat venenatis, dictum ex
                  nec, scelerisque dolor. Donec sit amet velit vel erat semper
                  malesuada sed lobortis libero.
                </p>
                <p>
                  Pellentesque quam metus, rutrum quis urna nec, eleifend rutrum
                  erat. Morbi at purus sed turpis faucibus pellentesque
                  consectetur a mi. Mauris malesuada tortor non lacus ultricies,
                  vel lacinia leo consequat. Sed lacus est, egestas sed purus
                  vitae, iaculis sagittis erat. Proin a fringilla metus.
                  Suspendisse ut augue vitae urna pulvinar iaculis. Aliquam
                  tempor ex eu lorem aliquam, a tincidunt purus cursus. Donec
                  vulputate lacus sapien, at varius turpis sollicitudin ut.
                </p>
                <p>
                  Donec finibus, sapien blandit rutrum blandit, nunc purus
                  tincidunt lorem, at dignissim eros turpis et odio. Phasellus
                  ultricies dui tristique, fermentum arcu nec, iaculis lacus.
                  Proin ullamcorper felis vitae tellus dictum, nec lacinia nunc
                  dignissim. Nam vel nulla quis quam porttitor tempor.
                  Suspendisse semper leo leo, sit amet iaculis lorem vestibulum
                  ac. Aenean pharetra elit nec justo vehicula, nec pretium
                  libero euismod. Vivamus augue mi, varius vitae feugiat nec,
                  dapibus et justo. In accumsan et neque eget feugiat.
                  Suspendisse iaculis tortor malesuada fringilla scelerisque.
                </p>
                <p>
                  Suspendisse pulvinar est nec velit iaculis hendrerit. Sed
                  lobortis eget eros vitae interdum. Donec tempor odio varius
                  leo euismod pulvinar. Pellentesque habitant morbi tristique
                  senectus et netus et malesuada fames ac turpis egestas. Nunc
                  vitae magna sed sapien vulputate vestibulum. Vivamus rutrum
                  diam ac imperdiet pretium. Cras condimentum laoreet nisl, quis
                  auctor mauris rhoncus ullamcorper.
                </p>
              </Container>
            </div>
          </div>
        </div>
      </Container>
    </main>
  );
};

export default About;
