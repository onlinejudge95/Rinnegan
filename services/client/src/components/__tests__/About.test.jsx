import React from "react";
import { render, cleanup } from "@testing-library/react";

import About from "../About";

afterEach(cleanup);

it("renders about", () => {
  const dom = render(<About />);
  expect(dom.getByText("Sentimental")).toHaveClass("title is-1");
});

it("renders", () => {
  const dom = render(<About />);
  expect(dom.asFragment()).toMatchSnapshot();
});
