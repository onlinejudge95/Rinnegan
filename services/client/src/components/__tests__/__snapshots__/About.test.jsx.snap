// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="title is-1"
    >
      Sentimental
    </h1>
    <hr />
    <br />
    <p
      class="content"
    >
      With the advent of Machine Learning, the filed of sentiment analysis has become somewhat trivial. You can find pre-made solutions to find sentiment of a word in a given text. We built upon this functionality and made this system.
    </p>
    <p
      class="content"
    >
      Sentimental is a platform where a user can come and serach for a 
      <strong>
        sentiment analysis
      </strong>
       on a given 
      <strong>
        keyword
      </strong>
      . After he has finalized his choice, he will be shown the analysis for that word by choosing the corpus from
      <strong>
        one or more of the social media platorms
      </strong>
      . Currently we support the following platforms to fetch data from
    </p>
    <ul
      class="content"
    >
      <li>
        Twitter
      </li>
    </ul>
    <p
      class="content"
    >
      As and when our infrastructure grows we will be using other platforms to include more diverse sentiments
    </p>
  </div>
</DocumentFragment>
`;
